### ISSUE TITLE
Why 2 steps User/Pass login for Hestia WebUI login

### ISSUE THREAD

**Issue by mehargags** [hestiacp/hestiacp#1071]
Hi Everyone,
Why have we done a 2 Step Wizard login form for Hestia's WebUI login ?  Is there a security factor implemented or was it just for aesthetics.  If it is just for the look and feel, I loved the old way of User/Pass in one go as for someone like me managing 100+ Servers, one click login is faster than clicking 3 Times (select user admin in autofill, Next, Next again on password.

Just my personal views... :)

**Comment by rmj-s** [comment]
Hi @mehargags,
I think this [commit](https://github.com/hestiacp/hestiacp/commit/e161b4a61694c78e626d9124b34c1b06267731a7) added extra load on servers due to continuously checking for 2FA config so they changed it to multistep. Yes, it is now a little bit time consuming to login, instead of multiple reloads the multi-step login should have been made with jQuery and Ajax.

**Comment by jaapmarcus** [comment]
We where able to push up the server load to 60/70% by manually guessing usernames/pressing random keys.
Also I had multiple issues with both systems with keychain (OsX) continuing to the "check" instead waiting for the answers. 

It also has the advantage hackers can't quickly guess usernames. 

I do agree it takes slightly longer and still minor improvements can be made.

**Comment by ScIT-Raphael** [comment]
@jaapmarcus and @rmj-s are both right, thank you guys for answering!

**Comment by mehargags** [comment]
For people running Hestia on Custom port and only allowing specific IPs access to that port the security wasn't an issue. That was the primary reason I asked for Hestia Port change as argument to Hestia Custom install command and a `v-change-sys-port` CLI command.

This is going to be a bit of a hindrance for admins managing multiple servers and having to open them multiple times a day. I guess we can do it in a better way or make it optional sometime later
